python/hyperonpy.cpp

```diff
@@ -110,6 +117,31 @@ exec_error_t *py_execute(const struct gnd_t* _cgnd, struct vec_atom_t* _args, st
     }
 }
 
+void py_match_(const struct gnd_t *_gnd, const struct atom_t *_atom, lambda_t_binding_array_t callback, void *context) {
+    py::object hyperon = py::module_::import("hyperon");
+    py::function call_match_on_grounded_atom = hyperon.attr("call_match_on_grounded_atom");
+    py::object pyobj = static_cast<GroundedObject const *>(_gnd)->pyobj;
+    CAtom catom = (atom_t *)_atom;
```

Comment:
We cannot wrap the atom owned by Rust into the Python object as it will be freed second time by Python object:
```suggestion
    CAtom catom = atom_clone(_atom);
```